themes/_core/js/product.js

```diff
@@ -172,7 +172,7 @@ function updateProduct(event, eventType, updateUrl) {
         ajax: 1,
         action: 'refresh',
         quantity_wanted:
-          eventType === 'updatedProductCombination' ? $quantityWantedInput.attr('min') : $quantityWantedInput.val(),
+          eventType === 'updatedProductCombination' ? 1 : $quantityWantedInput.val(),
```

Comment:
What if the combination has a minimum to order different than 1 ?

Reply:
L175 is only when you click to a new combination on the product page. The success data give the good minimum quantity of the combination.
And for keyup and keydown, it's not l175.

Where is the bug ? see #14744